src/pages/patientView/copyNumberAlterations/column/CohortColumnFormatter.tsx

```diff
@@ -2,18 +2,34 @@ import * as React from 'react';
 import {DiscreteCopyNumberData} from "shared/api/generated/CBioPortalAPI";
 import {CopyNumberCount} from "shared/api/generated/CBioPortalAPIInternal";
 import FrequencyBar from "shared/components/cohort/FrequencyBar";
+import DefaultTooltip from "shared/components/DefaultTooltip";
+
+export interface IGisticSummary {
+    amp: boolean;
+    qValue:number;
+    peakGeneCount:number;
+}
+
+export interface IGisticData {
+    [entrezGeneId:string]: IGisticSummary[];
+}
 
 export default class CohortColumnFormatter
 {
     public static renderFunction(data:DiscreteCopyNumberData,
-                                 copyNumberCountData:CopyNumberCount[])
+                                 copyNumberCountData:CopyNumberCount[],
+                                 gisticData:IGisticData)
     {
         const copyNumberCount = CohortColumnFormatter.getCopyNumberCount(data, copyNumberCountData);
         const freqViz = CohortColumnFormatter.makeCohortFrequencyViz(data, copyNumberCount);
+        const gisticValue = CohortColumnFormatter.getGisticValue(data, gisticData);
+        const gisticIcon = gisticValue === null ?
+            null : CohortColumnFormatter.makeGisticIcon(gisticValue.qValue, gisticValue.peakGeneCount);
 
         return (
             <div>
                 {(freqViz !== null) && freqViz}
+                {(gisticIcon !== null) && gisticIcon}
```

Comment:
Just followed the convention in the mutation table's Cohort column formatter here. Would it actually be better to use `<If>`?